solidity/package-lock.json

```diff
@@ -11427,6 +11427,27 @@
           "requires": {
             "has-flag": "^3.0.0"
           }
+        },
+        "ts-node": {
```

Comment:
Is this dependency needed now?

Reply:
It was there before, the order has just changed.